.circleci/config.yml

```diff
@@ -106,10 +98,9 @@ executors:
   docker:
     working_directory: /opt/app-root/apicast
     docker:
-    - image: docker:stable
+    - image: docker:23.0.2-cli-alpine3.17
```

Comment:
as in, docker stable is not good enough?

Reply:
`docker:stable` does not implement docker V2 (there is not `docker compose` command)